### PR TITLE
CT tests

### PR DESCRIPTION
**Pull request by c-bik**

### CODE REVIEW

**Comment by walter-weinmann** [comment]
Fixed: Typo : skvh.

**Comment by walter-weinmann** [comment]
Fixed: Refactoring group startup and teardown functions.

File: test/imem_SUITE.erl [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+        {timetrap, {minutes, 10}}
+    ].
+
+%%--------------------------------------------------------------------
+%% Suite related setup and teardown functions.
+%%--------------------------------------------------------------------
+
+init_per_suite(Config) ->
+    ct:pal(info, ?MAX_IMPORTANCE, ?MODULE_STRING ++ ":init_per_suite/1 - Start ===>~n", []),
+    ?imem_test_setup,
```
Comment (c-bik) [inline test/imem_SUITE.erl]: Replace `?imem_test_setup` macro with something like:
```erlang
application:load(imem),
application:set_env(imem, mnesia_node_type, disc),
imem:start();
```

File: test/imem_SUITE.erl [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+    Config.
+
+end_per_suite(_Config) ->
+    ct:pal(info, ?MAX_IMPORTANCE, ?MODULE_STRING ++ ":end_per_suite/1 - Start ===>~n", []),
+    ?imem_test_teardown,
```
Comment (c-bik) [inline test/imem_SUITE.erl]: Replace `?imem_test_teardown` macro with
```erlang
imem:stop()
```

File: test/imem_SUITE.erl [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+    meta_concurrency/1,
+    meta_operations/1,
+    meta_partitions/1,
+    meta_preparations/1,
+    skvh_concurrency/1,
+    skvh_operations/1
+]).
+
+-define(NODEBUG, true).
+-include_lib("imem.hrl").
```
Comment (c-bik) [inline test/imem_SUITE.erl]: Delete above two lines if they can be made redundant.

File: test/imem_SUITE.erl (hunk 2 above)
Comment (walter-weinmann) [reply test/imem_SUITE.erl]: Fixed.

**Pull request merged**